style.css

```diff
@@ -438,3 +468,250 @@ a {
   margin-bottom: 30px;
   text-align: center;
 }
+
+/* The desktop design */
+@media (min-width: 600px) {
```

Comment:
- [ ] As already pointed out by the previous reviewer, please make sure your the value of the `min-width` for your media query is between `992px` and `1100px`. This is very important to have a very good desktop design.